src/github.rs

```diff
@@ -26,14 +46,14 @@ actix_web::HttpResponse::Found()
 }
 #[derive(serde::Deserialize)]
 pub struct RepoParams {
-owner:String,    
-repo: String,
+    github_likes:String
```

Comment:
We can pass `github_likes` multiple times as well, `/api/get-identities/?github_likes=ftd-lang/ftd&github_likes=ftd-lang/fpm` for example.

Reply:
Also lets change to `github_starred` terminology.